Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml

```diff
@@ -1421,33 +1266,155 @@ tasks:
         simple: n/a
     separatecontext: false
     continueonerrortype: ""
-    view: |-
-      {
-        "position": {
-          "x": 240,
-          "y": 580
-        }
-      }
+    view: "{\n  \"position\": {\n    \"x\": 260,\n    \"y\": 530\n  }\n}"
     note: false
     timertriggers: []
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-view: "{\n  \"linkLabelsPosition\": {\n    \"25_19_yes\": 0.6,\n    \"3_42_#default#\": 0.32,\n    \"3_9_yes\": 0.5,\n    \"4_11_yes\": 0.43,\n    \"4_42_#default#\": 0.31,\n    \"7_18_yes\": 0.5\n  },\n  \"paper\": {\n    \"dimensions\": {\n      \"height\": 2965,\n      \"width\": 2262.5,\n      \"x\": -440,\n      \"y\": -320\n    }\n  }\n}"
-inputs:
-- key: RemoteIP
-  value:
-    complex:
-      root: inputs
-      accessor: remoteip
-  required: false
-  description: IP address of the service.
-  playbookInputQuery:
+  '50':
+    continueonerrortype: ''
+    id: '50'
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - '2'
+    note: false
+    quietmode: 0
+    separatecontext: true
+    skipunavailable: false
+    task:
+      brand: ''
+      description: Given an IP address, this playbook enriches Azure Compute, Microsoft Graph User, and IAM information and outputs Azure Compute, Microsoft Graph User, and IAM information.
+      id: 552168a5-fe49-4238-8c68-ad255bf6262c
+      iscommand: false
+      name: Azure - Enrichment
+      playbookId: Azure - Enrichment
```

Comment:
note that no PB input (which should be `AzureIP`) is set here - assuming that you are passing the default value from the alert then...

Reply:
updated in b8c5553

Reply:
Cool - hope you tested and it works fine :)